qa/suites/rgw/lifecycle/centos_latest.yaml

```diff
@@ -0,0 +1 @@
+.qa/distros/supported/centos_latest.yaml
```

Comment:
is there a reason to require centos here? ideally we would use a random supported distro instead. for example, rgw/crypt has this:
```
qa/suites/rgw/crypt $ ln -s .qa/distros/supported-random-distro$ .
```
the $ tells teuthology-suite to select a random yaml from the ceph/qa/distros/supported-random-distro\$ directory

Reply:
done.